### ISSUE TITLE
Lines should be easier to select on the map

### ISSUE THREAD

**Issue by dakotabenjamin** [mapbox/geojson.io#579]
When you have a line on the map it is very finicky, there are maybe one or two pixels you have to get your cursor on to select the line. This is not breaking, but a little aggravating to work with.

**Comment by tomscholz** [comment]
Do you have any idea how that could be improved?

**Comment by dakotabenjamin** [comment]
Probably not is a simple way. I know with Turf.js you can do distance from line calcs on the fly, so that could be used to select the line with a certain threshold for detection. There could also be an invisible buffer, although that could prove unwieldy.

**Comment by tomscholz** [comment]
This issue reminds me of this: 
https://github.com/mapbox/geojson.io/pull/568
https://github.com/mapbox/geojson.io/issues/565

:)

**Comment by dakotabenjamin** [comment]
Yes it does. I'm new to a lot of web mapping development and it amazes me how much you can accomplish with CSS. I'll look into getting that PR working.

**Comment by tomscholz** [comment]
Sounds great 👍  :)

**Comment by AndrewSepic** [comment]
No longer relevant due to new application launch.
